eth/protocols/snap/sync.go

```
@@ -746,7 +750,12 @@ func (s *Syncer) loadSyncStatus() {
 						s.accountBytes += common.StorageSize(len(key) + len(value))
 					},
 				}
-				task.genTrie = trie.NewStackTrie(task.genBatch)
+				if s.zk {
+					zktrie, _ := trie.NewZkTrie(common.Hash{}, trie.NewZktrieDatabase(&readonlyKeyValueStore{memorydb.New(), task.genBatch}))
```

Is it okay to ignore the error returned?